src/thirdparty/tests/mfem_smoke.cpp

```diff
@@ -12,6 +12,19 @@
 #include "mfem.hpp"
 #include "gtest/gtest.h"
 
+//-----------------------------------------------------------------------------
+TEST(mfem_smoke, check_version)
+{
+  std::cout << "Using mfem version: "     //
+            << MFEM_VERSION_MAJOR << "."  //
+            << MFEM_VERSION_MINOR << "."  //
+            << MFEM_VERSION_PATCH << std::endl;
+
+  EXPECT_TRUE(MFEM_VERSION_MAJOR >= 0);
+  EXPECT_TRUE(MFEM_VERSION_MINOR >= 0);
+  EXPECT_TRUE(MFEM_VERSION_PATCH >= 0);
+}
+
```

Comment:
I was also planning to update to mfem@4.5.2 but it depends on raja@2022.10.* and we're currently on raja@2022.03.*

The issue is related to mfem's usage of `raja::launch`, which axom is not currently using, so I'd expect the update to be relatively straightforward.

Reply:
Make sure to use v2022.10.5. There are incremental changes from 2022.10.0 to 2022.10.5 to fix various build issues. It was an unfortunate time for our lack of comprehensive testing.

Reply:
I created an issue to track our TPL updates: https://github.com/LLNL/axom/issues/1093